.github/workflows/lint.jsonnet

```diff
@@ -73,10 +73,61 @@ local pre_commit_manual_job = {
   steps: [
     actions.checkout(),
     gha.git_safedir,
+
+    # We need to specify the hook alias
+    # "semgrep-docker-develop-from-semgrep-semgrep" as opposed to the
+    # hook id "semgrep-docker-develop" because we have multiple pre-commit
+    # hooks with this id, and we only want this specific alias to run
+    # for this job.
+    {
+      uses: 'pre-commit/action@v3.0.0',
+      with: {
+        extra_args: 'semgrep-docker-develop-from-semgrep-semgrep --hook-stage manual',
+      },
+    },
+  ],
+};
+
+
+// This is job has a different purpose than the dogfood job above.
+// This job tests that the semgrep/pre-commit repo containts a valid
+// pre-commit hook that users can install.
```

Comment:
Should we test this in semgrep/pre-commit instead? It sounds like this just depends on the code there, which depends on Docker/pip so it would seem to me that we would want this to run on a push to that repository and have updating that be part of the release flow here.

Reply:
Yeah, I think testing the hook in semgrep/pre-commit makes sense.

Now I'm thinking a bit more about the dogfood job above (pre_commit_manual_job), but that's a separate thing.

I'll change this back to draft and will likely just close it when the semgrep/pre-commit PR is merged.